app/api/srch/search.rb

```diff
@@ -34,7 +34,7 @@ class Search < Grape::API
         Search.execute(:profiles, params)
       end
 
-      # Request URL should be /api/srch/notes?srchString=QRY[&seq=KEYCOUNT&showCount=NUM_ROWS&pageNum=PAGE_NUM]
```

Comment:
Just curious what the reason for removing these parameters... sorry, haven't looked at this code for a while, can you remind me if they're not used or something?